schedule/scheduler/scheduler_test.go

```diff
@@ -216,9 +219,54 @@ func TestExtendSchedule(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "last shift user no longer in rotation",
+			input: &schedule.Schedule{
+				Shifts: []*schedule.Shift{
+					{
+						StartDate: time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+						User:      "first",
+					},
+					{
+						StartDate: time.Date(2020, 1, 2, 0, 0, 0, 0, time.UTC),
+						StopDate:  time.Date(2020, 1, 2, 0, 0, 0, 0, time.UTC),
+						User:      "second",
+					},
+				},
+			},
+			users:        []string{"first", "third"},
+			durationDays: 1,
+			stop:         time.Date(2020, 1, 5, 0, 0, 0, 0, time.UTC),
+			wantErr:      false,
+			want: &schedule.Schedule{
+				Shifts: []*schedule.Shift{
+					{
+						StartDate: time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+						User:      "first",
+					},
+					{
+						StartDate: time.Date(2020, 1, 2, 0, 0, 0, 0, time.UTC),
+						User:      "second",
```

Comment:
I'm probably being dumb...but why does `second` get scheduled if it's not in the `users` list anymore?

Reply:
This change does not modify the _existing_ schedule upon extension.

Reply:
Oh right sorry 🙈